package.json

```diff
@@ -1,14 +1,19 @@
 {
-  "name": "openhim-core-js",
+  "name": "openhim-core",
   "description": "The OpenHIM core application that provides logging and routing of http requests",
-  "version": "0.1.0",
+  "version": "1.1.0",
   "private": true,
```

Comment:
Yeah that's right. I'l switch this once we are ready to do a npm release. This prevents us from being able to release by accident before we are ready.